developments_build/06_qaqc.sh

```diff
@@ -1,81 +1,74 @@
 #!/bin/bash
 source config.sh
 
```

Comment:
I think this file can get deleted. We run our QAQC scripts as we populate the variables.